crates/sui-protocol-config/src/lib.rs

```diff
@@ -2121,6 +2123,12 @@ impl ProtocolConfig {
                         cfg.feature_flags.consensus_choice = ConsensusChoice::SwapEachEpoch;
                     }
                 }
+                45 => {
+                    // Use tonic networking for consensus, in tests and devnet.
+                    if chain != Chain::Testnet && chain != Chain::Mainnet {
+                        cfg.feature_flags.consensus_network = ConsensusNetwork::Tonic;
```

Comment:
is the plan to eventually switch all networks to this?

Reply:
Yes if things go well. I don't need the if condition for now, but we are not 100% sure if we will switch to tonic for mysticeti in testnet.